mod/node-api/handlers/utils/id.go

```diff
@@ -70,36 +70,37 @@ func SlotFromBlockID[StorageBackendT interface {
 	return storage.GetSlotByBlockRoot(root)
 }
 
-// SlotFromExecutionID returns a slot from the execution number ID.
+// ParentSlotFromTimestampID returns the parent slot corresponding to the
+// timestamp ID.
 //
-// NOTE: `executionID` shares the same semantics as `stateID`, with the
-// modification of being able to query by beacon block <executionNumber>
-// instead of <stateRoot>.
+// NOTE: `timestampID` shares the same semantics as `stateID`, with the
```

Comment:
nit: here and in the rest of the package I would use `timestamp` instead of `timestampID`